core/src/main/java/org/fao/geonet/kernel/metadata/DefaultStatusActions.java

```diff
@@ -210,13 +210,13 @@ public Set<Integer> onStatusChange(List<MetadataStatus> listOfStatus) throws Exc
     private void applyRulesForStatusChange(MetadataStatus status) throws Exception {
         String statusId = status.getStatusValue().getId() + "";
         if (statusId.equals(StatusValue.Status.APPROVED)) {
+            //            AccessManager accessManager = context.getBean(AccessManager.class);
```

Comment:
It's a bit unclear about this change, can be added a comment in the code to explain it?

Reply:
You are right, the method is not used. Checking the comments, it seems was implemented initially to link the approval/rejection with the publication/un-publication of the metadata, but that requirement was discarded.

I would remove that method, to avoid confusion.

https://github.com/geonetwork/core-geonetwork/blob/2fdfdcd9d33d7ac862e0675eb0e601e54a0c37a0/core/src/main/java/org/fao/geonet/kernel/metadata/DefaultStatusActions.java#L199-L209

Reply:
Removed